Library/Homebrew/cmd/update-report.rb

```diff
@@ -249,8 +249,21 @@ def updated?
   def migrate_tap_migration
     report[:D].each do |full_name|
       name = full_name.split("/").last
-      next unless (dir = HOMEBREW_CELLAR/name).exist? # skip if formula is not installed.
       next unless new_tap_name = tap.tap_migrations[name] # skip if formula is not in tap_migrations list.
+      if tap == "caskroom/cask"
+        next unless (HOMEBREW_REPOSITORY/"Caskroom"/name).exist?
+        new_tap = Tap.fetch(new_tap_name)
+        new_tap.install unless new_tap.installed?
+        system HOMEBREW_BREW_FILE, "install", name
+        system HOMEBREW_BREW_FILE, "link", name, "--force"
+        ohai "#{name} has been moved to Homebrew. Installing #{name}...", <<-EOS.undent
+          To uninstall the cask run:
+            brew cask uninstall --force #{name}
+        EOS
```

Comment:
Thanks for addressing the problem with `ohai`'s arguments, but now this is somewhat awkwardly output *after* the formula has already been installed and linked. Shouldn't this be placed before the preceding two lines?